### PR TITLE
Support backend sorting

### PR DESCRIPTION
**Pull request by MingoLi**
[US136564](https://rally1.rallydev.com/#/?detail=/userstory/627870441983&fdp=true): Support sortable last name

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-mapping-table.js [hunk 1]
```diff
@@ -302,15 +298,19 @@ class ActivityEvaluatonMappingTable extends RtlMixin(LocalizeEvaluationMixin(Mob
 		`;
 	}
 
-	_renderSortButton(field, btName) {
-		const noSort = this._sortField !== field;
+	_renderSortButton() {
```
Comment (alexvlsem) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-mapping-table.js:301]: May be rename it to plural?

**Review by alexvlsem — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-mapping-table.js (hunk 1 above)
Comment (MingoLi) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-mapping-table.js:301]: good catch! I will make all the changes once I have done the performance tests

**Review by alexvlsem — COMMENTED** [review]
Is it possible to update goldens for our project only?

**Comment by MingoLi** [comment]
> 

I don't know to be honest, but I observed the same visual diff error has been happening for other PR as well. We can potentially wait for it being fixed in master

**Review by alexvlsem — APPROVED** [review]

**Review by jmurray-d2l — APPROVED** [review]

**Pull request merged**